diff -urN -x .git -x '*.md' /home/cwp/rpmbuild/BUILD/das2C-2.3-pre4/makefiles/Linux.mak /home/cwp/git/das2C/makefiles/Linux.mak
--- makefiles/Linux.mak	2021-11-12 15:07:32.000000000 -0600
+++ makefiles/Linux.mak	2021-11-28 15:08:49.814564334 -0600
@@ -70,11 +70,11 @@
 
 UTIL_OBJS= $(patsubst %,$(BD)/%.o,$(UTIL_PROGS))
 
-INST_HDRS= $(patsubst %.h,$(INST_INC)/das2/%.h,$(HDRS))
+INST_HDRS= $(patsubst %.h,$(DESTDIR)$(INST_INC)/das2/%.h,$(HDRS))
 
 BUILD_UTIL_PROGS= $(patsubst %,$(BD)/%, $(UTIL_PROGS))
 
-INST_UTIL_PROGS= $(patsubst %,$(INST_NAT_BIN)/%, $(UTIL_PROGS))
+INST_UTIL_PROGS= $(patsubst %,$(DESTDIR)$(INST_NAT_BIN)/%, $(UTIL_PROGS))
 
 BUILD_TEST_PROGS = $(patsubst %,$(BD)/%, $(TEST_PROGS))
 
@@ -112,19 +112,19 @@
 	$(CC) $(CTESTFLAGS) $< $(BD)/$(TARG).a $(LFLAGS) -o $@ 
 
 # Pattern rule for installing static libraries
-$(INST_NAT_LIB)/%.a:$(BD)/%.a
+$(DESTDIR)$(INST_NAT_LIB)/%.a:$(BD)/%.a
 	install -D -m 664 $< $@
 	
 # Pattern rule for installing dynamic libraries
-$(INST_NAT_LIB)/%.so:$(BD)/%.so
+$(DESTDIR)$(INST_NAT_LIB)/%.so:$(BD)/%.so
 	 install -D -m 775 $< $@	
 
 # Pattern rule for installing library header files
-$(INST_INC)/das2/%.h:das2/%.h
+$(DESTDIR)$(INST_INC)/das2/%.h:das2/%.h
 	install -D -m 664 $< $@	 
 
 # Pattern rule for installing binaries
-$(INST_NAT_BIN)/%:$(BD)/%
+$(DESTDIR)$(INST_NAT_BIN)/%:$(BD)/%
 	install -D -m 775 $< $@	 
 	
 
@@ -192,7 +192,7 @@
 # Install everything
 install:lib_install $(INST_UTIL_PROGS)
 
-lib_install:$(INST_NAT_LIB)/$(TARG).a $(INST_NAT_LIB)/$(TARG).so $(INST_HDRS)
+lib_install:$(DESTDIR)$(INST_NAT_LIB)/$(TARG).a $(DESTDIR)$(INST_NAT_LIB)/$(TARG).so $(INST_HDRS)
 
 # Does not install static object that that it can be used with proprietary
 # software
